enterprise/models.py

```diff
@@ -2508,3 +2508,44 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class EnterpriseAnalyticsUser(TimeStampedModel):
+    """
+    Model for analytics users.
+
+     .. no_pii:
+    """
+    enterprise_customer_user = models.ForeignKey(
+        EnterpriseCustomerUser,
+        blank=False,
+        null=False,
+        on_delete=models.CASCADE
+    )
+    analytics_user_id = models.CharField(
+        max_length=255,
+        blank=False,
```

Comment:
same as above, blank null is False by default.

Reply:
no harm in being explicit